src/functions/uploadToRSIS/application/batch-processor.ts

```diff
@@ -87,8 +87,9 @@ export async function processResult(config: Config, connection: Connection | und
 
     resultUpload.testResult.journalData.applicationReference;
     const appRef = formatApplicationReference(resultUpload.testResult.journalData.applicationReference);
+    const category = resultUpload.testResult.category;
```

Comment:
possibly use object destructuring for category instead of creating a const i.e.

const { category } = resultUpload.testResult;

not something i think necessarily needs a change but just to avoid duplicating the word

Reply:
changed